pytest/tests/replay/replay.py

```diff
@@ -0,0 +1,161 @@
+from account import Account
+from collections import OrderedDict
+from key import Key
+from messages.tx import *
+from messages.crypto import AccessKey, crypto_schema, PublicKey, Signature
+from messages.bridge import bridge_schema
+from serializer import BinarySerializer
+import mocknet_helpers
+
+import argparse
+import base58
+import base64
+import hashlib
+import json
+import os
+
+def generate_new_key():
+    return Key.implicit_account()
+
+def save_genesis_with_new_key_pair(genesis_path, key_pair, output_path):
+    NODE0_DIR = os.path.join(output_path, 'node0/')
+    if not os.path.exists(NODE0_DIR):
+        os.makedirs(NODE0_DIR)
+    with open(genesis_path) as fin:
+        genesis = json.load(fin)
+
+    some_validator_id = None
+    new_key = key_pair.pk.split(':')[1] if ':' in key_pair.pk else key_pair.pk
+    for validator in genesis['validators']:
+        validator['public_key'] = new_key
+        some_validator_id = validator['account_id']
+    for record in genesis['records']:
+        if 'AccessKey' in record:
+            record['AccessKey']['public_key'] = new_key
+    with open(os.path.join(NODE0_DIR, 'genesis.json'), 'w') as fout:
+        json.dump(genesis, fout, indent=2)
+
+    key_pair.account_id = some_validator_id
+    key_json = dict()
+    key_json['account_id'] = some_validator_id
+    key_json['public_key'] = key_pair.pk
+    key_json['secret_key'] = key_pair.sk
+    with open(os.path.join(NODE0_DIR, 'node_key.json'), 'w') as fout:
+        json.dump(key_json, fout, indent=2)
+    with open(os.path.join(NODE0_DIR, 'validator_key.json'), 'w') as fout:
+        json.dump(key_json, fout, indent=2)
+
+def prompt_to_launch_localnet():
+    input('Please launch your localnet node now and press enter to continue...')
```

Comment:
consider printing a command line that a user should run (possibly including the directory)